docs/iris/src/whatsnew/latest.rst

```diff
@@ -13,193 +13,255 @@ This document explains the changes made to Iris for this release
+  :class:`~iris.coords.CellMeasure`. (:pull:`3546`)
 
-* A :class:`~iris.coords.CellMeasure` requires a string ``measure`` attribute
-  to be defined, which can only have a value of ``area`` or ``volume``.
+* `Bill Little`_ set a :class:`~iris.coords.CellMeasure`'s
+  ``measure`` attribute to have a default value of ``area``.
   Previously, the ``measure`` was provided as a keyword argument to
-  :class:`~iris.coords.CellMeasure` with an default value of ``None``, which
-  caused a ``TypeError`` when no ``measure`` was provided.  The default value
-  of ``area`` is now used.
+  :class:`~iris.coords.CellMeasure` with a default value of ``None``, which
+  caused a ``TypeError`` when no ``measure`` was provided, since ``area`` or
+  ``volume`` are the only accepted values. (:pull:`3533`)
 
-* **All** plot types in `iris.plot` now use `matplotlib.dates.date2num
+* `Martin Yeo`_ set **all** plot types in `iris.plot` to now use
+  `matplotlib.dates.date2num
   <https://matplotlib.org/api/dates_api.html#matplotlib.dates.date2num>`_
   to format date/time coordinates for use on a plot axis (previously
   :meth:`~iris.plot.pcolor` and :meth:`~iris.plot.pcolormesh` did not include
-  this behaviour).
+  this behaviour). (:pull:`3762`)
 
-* Date/time axis labels in `iris.quickplot` are now **always** based on the
-  ``epoch`` used in `matplotlib.dates.date2num
+* `Martin Yeo`_ changed date/time axis labels in `iris.quickplot` to now
+  **always** be based on the ``epoch`` used in `matplotlib.dates.date2num
   <https://matplotlib.org/api/dates_api.html#matplotlib.dates.date2num>`_
   (previously would take the unit from a time coordinate, if present, even
   though the coordinate's value had been changed via ``date2num``).
+  (:pull:`3762`)
 
-* Attributes of cell measures in NetCDF-CF files were being discarded during
-  loading. They are now available on the :class:`~iris.coords.CellMeasure` in
-  the loaded :class:`~iris.cube.Cube`. See :pull:`3800`.
+* `Patrick Peglar`_ newly included attributes of cell measures in NETCDF-CF
+  file loading; they were previously being discarded. They are now available on
+  the :class:`~iris.coords.CellMeasure` in the loaded :class:`~iris.cube.Cube`.
+  (:pull:`3800`)
 
 
 Incompatible Changes
 ====================
 
-* The method :meth:`~iris.cube.CubeList.extract_strict`, and the ``strict``
+* `Patrick Peglar`_ rationalised :class:`~iris.cube.CubeList` extraction
+  methods:
+
+  The method :meth:`~iris.cube.CubeList.extract_strict`, and the ``strict``
   keyword to :meth:`~iris.cube.CubeList.extract` method have been removed, and
   are replaced by the new routines :meth:`~iris.cube.CubeList.extract_cube` and
   :meth:`~iris.cube.CubeList.extract_cubes`.
   The new routines perform the same operation, but in a style more like other
-  ``Iris`` functions such as :meth:`~iris.load_cube` and :meth:`~iris.load_cubes`.
+  ``Iris`` functions such as :meth:`~iris.load_cube` and:meth:`~iris.load_cubes`.
```

Comment:
There should be a space between "and" and ":meth:"

Reply:
d1038c403d23de9e461e3fe5e3a94cb7def73778